Client/Frontend/Autofill/CreditCard/CreditCardSettingsView/CreditCardInputView.swift

```diff
@@ -17,6 +23,7 @@ struct CreditCardEditView: View {
     @State var removeButtonColor: Color = .clear
     @State var borderColor: Color = .clear
     @State var textFieldBackgroundColor: Color = .clear
+    @State private var text = CreditCardInputText()
```

Comment:
```suggestion
    @State private var inputText = CreditCardInputText()
```

Text is a SwiftUI view as well so better to name it something more verbose